_posts/2022-04-11-1.15-announcement.md

```diff
@@ -0,0 +1,431 @@
+some of the features that make taking checkpoints so fast. In some cases like 
+resuming from a retained checkpoint in which the checkpoint is somehow considered a 
+a savepoint, it is unclear to the user when they can actually clean it up. 
+
+With [FLIP-193 (Snapshots ownership)](https://cwiki.apache.org/confluence/display/FLINK/FLIP-193%3A+Snapshots+ownership) 
+the community aims to make the ownership the only difference between savepoint and 
+checkpoint. In the 1.15 release the community has fixed some of those shortcomings 
+by supporting 
+[native and incremental savepoints](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/ops/state/savepoints/#savepoint-format). 
+Savepoints always used to use the 
+canonical format which made them slower. Also writing full savepoints for sure takes 
+longer than doing it in an incremental way. With 1.15 if users use the native format 
+to take savepoints as well as the RocksDB state backend, savepoints will be 
+automatically taken in an incremental manner. The documentation has also been 
+clarified to provide a better overview and understanding of the differences between 
+checkpoints and savepoints. The semantics for 
+[resuming from savepoint/retained checkpoint](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/ops/state/savepoints/#resuming-from-savepoints) 
+have also been clarified introducing the CLAIM and NO_CLAIM mode. With 
+the CLAIM mode Flink takes over ownership of an existing snapshot, with NO_CLAIM it 
+creates its own copy and leaves the existing one up to the user. Please note that 
+NO_CLAIM mode is the new default behavior. The old semantic of resuming from 
+savepoint/retained checkpoint is still accessible but has to be manually selected by 
+choosing LEGACY mode.
+
+
+### Elastic scaling with reactive mode and the adaptive scheduler
+
+Driven by the increasing number of cloud services built on top of Apache Flink, the 
+project is increasingly cloud native which makes elastic 
+scaling more and more important. 
+
+This release improves metrics for the [reactive mode](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/elastic_scaling/#reactive-mode), which is a job-scope mode where the JobManager will try to use all TaskManager resources available. To do this, we made all the metrics in 
+the Job scope work correctly when reactive mode is enabled 
+([yes, only limitations have been removed from the documentation](https://github.com/apache/flink/pull/17766/files)). 
+
+We also added an exception history for the [adaptive scheduler](https://cwiki.apache.org/confluence/display/FLINK/FLIP-160%3A+Adaptive+Scheduler), which is a new scheduler that first declares the required resources and waits for them before deciding on the parallelism with which to execute a job. 
+
+Furthermore, downscaling is sped up by 10x. The TaskManager now has a dedicated 
```

Comment:
I fear that we are promising too much here - this downscaling speed probably does not include state restore,...for the new job getting started at a lower parallelism...